app/javascript/configureStore.js

```diff
@@ -0,0 +1,30 @@
+import { createStore, applyMiddleware } from "redux";
+import thunk from "redux-thunk";
+
+const initialState = {
+  things: [
+    {
+      name: "test",
+      guild: "123"  
+    }
+  ]
+};
+
+function rootReducer(state, action) {
```

Comment:
[OPTIONAL] - Using ES6 syntax makes your code clean. You practice using an arrow `function` more.